README.md

```diff
@@ -53,6 +53,9 @@ new Siema({
   loop: false,
   onInit: function(){},
   onChange: function(){},
+  afterPrev: function() {},
+  afterNext: function() {},
+  onDestroy: function() {}  
```

Comment:
We have to ditch `afterPrev()` and `afterNext()` from this PR. In current implementation it invokes actually straight before `next()` and `prev()`, not after it. Doing before and after is a bit more complicated as it needs to listen for `transitionstart` and `transitionend` events on instance's `sliderFrame`. Let's keep it for future versions of Siema.

The `onDestory()` callback — perfecto :)

Don't worry about changing that, I will sort it out dude.